cmd/thanos/tools.go

```diff
@@ -55,7 +55,6 @@ func checkRulesFiles(logger log.Logger, patterns *[]string) error {
 		if err != nil || matches == nil {
 			err = errors.New("matching file not found")
 			level.Error(logger).Log("result", "FAILED", "error", err)
-			level.Info(logger).Log()
```

Comment:
Do we need to remove this?

Reply:
These are just empty logs, Its output is - `level=info`, So I removed all `level.Info(logger).Log()`

Reply:
I think it might be intended to leave some spaces between lines.

Reply:
Ok, Should I reintroduce all instances of `level.Info(logger).Log()` throughout the entire code, or only in this specific part of the code?

Reply:
I don't have strong preference. I am ok either way to keep them or remove them.